tests/test_report_generator.py

```diff
@@ -86,3 +87,41 @@ def test_report_gen(test_dictionary):
         assert result.filename == output_file_path
         # Validate the page size of result
         assert result.pagesize == (612.0, 792.0)
+
+
+@patch("tpt_reports.report_generator.utils.ImageReader")
+@patch("tpt_reports.report_generator.Image")
+class TestGetImage(TestCase):
+    """Test get_image clase."""
```

Comment:
Fix typo and improve wording:
```suggestion
    """Tests for the get_image function."""
```